paddle/majel/ddim/ddim.cc

```diff
@@ -1,4 +1,4 @@
-#include "paddle/majel/ddim.h"
+#include "paddle/majel/ddim/ddim.h"
```

Comment:
I agree that current CMake rule in Paddle majel with `cc_library()` and `cc_test()` has made module describing quite clearly, therefore simple module like `ddim` and `place` is not necessery to moved into a directory. However, if a module is really complicated, we should still gather related files up in a subdirectory.

This is my personal understanding, I'm not sure whether it is right ...